cookbook/websocket.rst

```diff
@@ -0,0 +1,47 @@
+How to configure Websocket?
+===========================
+
+Sulu comes with a sulu-wide websocket application. This application is optional
+but comes with a few feature which can only be used if the websocket is running.
+
+For example the real-time notification for :doc:`../bundles/collaboration`
+can only be displayed when the websocket is running.
+
+Configuration
+-------------
+
+If you use the config layout of `sulu-standard
+<https://github.com/sulu-io/sulu-standard>`_ you only have to set following
+parameters.
+
+.. list-table::
+
+    * - websocket_url
+      - One url where the websocket is reachable.
+    * - websocket_port
+      - The port where thewebsocket should listen on.
+
+If you have multiple installations of sulu on a single server be sure that you
+use unique ports per installation.
+
+if you have you own application you can configure the url and port in the
+following config.
+
+.. code-block:: yaml
+
+    sulu_websocket:
+        server:
+            http_host: %websocket_url%
+            port: %websocket_port%
+
+.. warning::
+
+    Because of a bug in the current sulu version you also have to set the
+    ``app/config/admin/config.yml`` ``sulu_content: preview: websocket: true``.
```

Comment:
That's not a common description... I would just write that you have to set the value of `sulu_content.preview.websocket` to `true`. It also doesn't really matter in which file this is done.